tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -590,14 +590,20 @@ class SystemMemoryManager {
         }
     }
 
-    void fetch_queue_write(uint32_t command_size_B, const uint8_t cq_id) {
+    void fetch_queue_write(uint32_t command_size_B, const uint8_t cq_id, bool stall_prefetcher = false) {
```

Comment:
can you add a comment about relationship of setting stall_prefetcher and executing a trace